src/mcts/search.cc

```diff
@@ -250,7 +250,10 @@ std::vector<std::string> Search::GetVerboseStats(Node* node,
         << ") ";
 
     oss << "(Q+U: " << std::setw(8) << std::setprecision(5)
-        << edge.GetQ(fpu) + edge.GetU(U_coeff) << ") ";
+        << (edge.GetN() == 0 ? fpu :
+            (params_.GetLogitQEnabled() ? FastLogit(0.99999994f * edge.GetQ(0)) :
+             edge.GetQ(0))) + edge.GetU(U_coeff)
```

Comment:
(!params_.GetLogitQEnabled() || edge.getN() == 0 ? edge.GetQ(fpu)  : FastLogit(0.99999994f * edge.GetQ(fpu))) + edge.GetU(U_coeff)

Produces the identical output and is simpler - no nested ternary.
Separately it would be simpler if this logic just migrated into edge.GetQ(fpu, logit_enabled) - would avoid repetition in the 3 places of the constant.